### PR TITLE
Add Contributing and Code of Conduct

### PR DESCRIPTION
**Pull request by the-codingninja**
@salman-bhai Please Review

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+**Contributions**
```
Comment (sbshah97) [inline CONTRIBUTING.md]: Put (##) instead.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+
+- A problem needs to be selected from [Project Euler](https://projecteuler.net/archives)
+
+- A folder should be made for the seperate problem
```
Comment (sbshah97) [inline CONTRIBUTING.md]: Separate**

**Review by sbshah97 — CHANGES_REQUESTED** [review]
Make the requested changes and send it in. I'll merge it.

File: CONTRIBUTING.md (hunk 2 above)
Comment (the-codingninja) [reply CONTRIBUTING.md]: What is to replaced here @salman-bhai

Comment (the-codingninja) [reply CONTRIBUTING.md]: Okay got it @salman-bhai

**Comment by the-codingninja** [comment]
https://github.com/salman-bhai/project-euler/issues/6 @salman-bhai Please review

**Comment by the-codingninja** [comment]
https://github.com/salman-bhai/project-euler/issues/7#issuecomment-333353012 @salman-bhai review

**Review by sbshah97 — APPROVED** [review]
Looks good! Good work on this @ms10398 !

**Comment by the-codingninja** [comment]
Thanks @salman-bhai

**Pull request merged**